src/lib/RangeHelper.js

```diff
@@ -385,6 +385,82 @@ define(function (require) {
 			return '';
 		};
 
+
+		/**
+		 * Replace elements with provided selector inside user selection with
+		 * their children
+		 *
+		 * This method provides an easier way to have the same final aspect
+		 * for HTML as powerful as the browser's native execCommand() method
+		 * when removing HTML tag formatting.
+		 *
+		 * This method is mostly meant to be used with <b>inline</b> HTML
+		 * Elements. Having it working with block level elements is not
+		 * guaranteed.
+		 *
+		 *
+		 * @param {String} removeElementSelector Selector match what to delete
+		 * @return undefined
+		 * @function
+		 * @name splitAtSelection
+		 * @memberOf RangeHelper.prototype
+		 * @author brunoais
+		 */
+		base.splitAtSelection = function (removeElementSelector) {
```

Comment:
Thinking about it, being able to remove elements only from the selection could be really useful! It might be worth splitting this into two methods, `splitAtSelection()` and `removeFromSelection()` (or some better name if you can think of it).